corehq/apps/app_manager/_design/views/saved_app/map.js

```diff
@@ -20,7 +20,8 @@ function(doc){
             comment_from: doc.comment_from,
             is_released: doc.is_released,
             case_sharing: doc.case_sharing,
-            build_profiles: doc.build_profiles
+            build_profiles: doc.build_profiles,
+            vellum_case_management: !!doc.vellum_case_management
```

Comment:
Yeah, it'll reindex the app db. Knockout uses the new property: https://github.com/dimagi/commcare-hq/pull/12387/files#diff-0e1fa007e17c990453d2c608a2de55a9R313